test/fx2trt/core/test_import_fx2trt.py

```diff
@@ -5,7 +5,7 @@
 
 # Test that this import should not trigger any error when run
 # in non-GPU hosts, or in any build mode.
-import torch.fx.experimental.fx2trt.lower as fxl  # noqa: F401
+import fx2trt_oss.fx.lower as fxl  # noqa: F401
```

Comment:
For my understanding—is the stuff in `test/fx2trt` eventually supposed to go as well?

Reply:
Yes. They will need to go. I'm actually surprised that they are here.